policy/blacklist.go

```diff
@@ -1,1027 +0,0 @@
-// Copyright 2014 ISRG.  All rights reserved
```

Comment:
Original plan was to move the PSL out as a follow-up to this PR so that the change set doesn't get too bloated.